src/index.html

```diff
@@ -81,6 +81,13 @@
         };
         window.document.body.appendChild(closeBtn);
       }
+
```

Comment:
FYI: It seems like this code only functions locally, and the index.html file could be removed. The widget demo site already duplicates this code, and it’s not currently live anywhere. It could still be useful for local development, but that’s the extent of its utility.

Reply:
agreed!  I did find it useful for local development so I thought it worth leaving it in there, and might as well update it to match the demo and allow exercising the new function we added.